tests/profiler/pytorch/test_pytorch_profiler.py

```diff
@@ -86,3 +86,5 @@ def test_pytorch_profiler(pytorch_profiler_config_parser, out_dir):
         assert len(events) == 386
     elif is_pt_1_6():
         assert len(events) == 672
+    elif is_pt_1_7():
+        assert 200 <= len(events) <= 555
```

Comment:
Does the number of events actually vary from 200 to 555? If so, is there a reason why? This seems like a pretty large gap.

Reply:
I was just testing it. Fixed in the code. It should be greater than 220.